pyiceberg/io/fsspec.py

```diff
@@ -86,6 +88,10 @@ def s3v4_rest_signer(properties: Properties, request: AWSRequest, **_: Any) -> A
         raise SignError("Signer set, but token is not available")
 
     signer_url = properties.get(S3_SIGNER_URI, properties["uri"]).rstrip("/")
+    signer_endpoint = properties.get(S3_SIGNER_ENDPOINT, None)
+    if signer_endpoint is None:
+        signer_endpoint = properties.get("endpoint", S3_SIGNER_ENDPOINT_DEFAULT_VALUE)
```

Comment:
I'm not sure if I understand what this "endpoint" property key is. Is there any reason why it shouldn't be something like this: `signer_endpoint = properties.get(S3_SIGNER_ENDPOINT, S3_SIGNER_ENDPOINT_DEFAULT_VALUE)`?